ssl/statem/statem_clnt.c

```diff
@@ -2126,7 +2140,7 @@ WORK_STATE tls_post_process_server_certificate(SSL_CONNECTION *s,
     }
 
     if ((clu = ssl_cert_lookup_by_pkey(pkey, &certidx,
-				       SSL_CONNECTION_GET_CTX(s))) == NULL) {
+                       SSL_CONNECTION_GET_CTX(s))) == NULL) {
```

Comment:
Why this change? It looked correct before?

Reply:
I'm...not sure, clearly an error on my part.